src/components/side-panel/interaction-panel/tooltip-config/tooltip-chicklet.js

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React, {Component} from 'react';
+import styled from 'styled-components';
+import {ChickletButton, ChickletTag} from 'components/common/item-selector/chickleted-input';
+import {Hash, Delete} from 'components/common/icons';
+import DropdownList from 'components/common/item-selector/dropdown-list';
+import {Tooltip} from 'components/common/styled-components';
+import {FormattedMessage} from 'react-intl';
+import onClickOutside from 'react-onclickoutside';
+import {FIELD_OPTS} from 'constants/default-settings';
+import {TOOLTIP_FORMATS} from 'constants/tooltip';
+
+function getFormatLabels(fields, fieldName) {
+  const fieldType = fields.find(f => f.name === fieldName).type;
+  const formatLabels = (FIELD_OPTS[fieldType].format.tooltip || []).map(v => v.label);
+  formatLabels.unshift(TOOLTIP_FORMATS.NONE.label);
+  return formatLabels;
```

Comment:
should pass the array of formats object instead of just the label.

You can use the `displayOption` props in `DropdownList` and pass `item => item.label`. This way, you can save the  `format.type` e.g. `TOOLTIP_FORMAT_TYPES.DECIMAL` in fieldsToShow.format instaed of the label. Saving the label is dangerous, because we can change the label at any time to make it more descriptive, but it will break people's maps when they load it back